### ISSUE TITLE
Data augmentation in the EEG domain

### ISSUE THREAD

**Issue by cwbeitel** [projectclarify/clarify#126]
Data augmentation in the visual domain commonly takes the form of alterations to color, contrast, brightness, and noise as well as the addition of black patches, shifts, flips, skew, and so on. Such augmentations help prevent model over-fitting / help generalization beyond the training data-set. We have an opportunity to consider these by analogy in the EEG domain.

The addition of black patches (whether by channel or region) or gaussian noise have natural analogs. But what about color channel dropout? Might we benefit from dropping out specific wave patterns or other features identified following raw signal processing? What augmentation can be performed to avoid over-fitting to one experiment's specific choice of electrode placement?

This is related to #75 and #124 when the learning problem is reconstruction, for example, but is a subset of the broader space of SSL methods.
